client/crypto-sdk/lib/address.cpp

```diff
@@ -14,18 +15,34 @@
 using namespace std;
 using namespace Orbs;
 
+const uint8_t Address::MAIN_NETWORK_ID = 0x14; // "M" in BASE58.
+const uint8_t Address::TEST_NETWORK_ID = 0x1A; // "T" in BASE58;
+
 const uint8_t Address::VERSION = 0;
 const uint32_t Address::PUBLIC_KEY_SIZE = 32;
-const uint32_t Address::VERSION_FIELD_SIZE = 1;
+const uint32_t Address::NETWORK_ID_SIZE = 1;
+const uint32_t Address::VERSION_SIZE = 1;
+const uint32_t Address::VIRTUAL_CHAIN_ID_SIZE = 3;
+const uint8_t Address::VIRTUAL_CHAIN_ID_MSB = 0x08;
 const uint32_t Address::ACCOUNT_ID_SIZE = 20;
 const uint32_t Address::CHECKSUM_SIZE = 4;
-const uint32_t Address::ADDRESS_LENGTH = 34;
+const uint32_t Address::ADDRESS_LENGTH = 39;
```

Comment:
Please comment on how you got to these numbers to keep it maintainable

Reply:
Done